assets/js/modules/pagespeed-insights/components/dashboard/DashboardPageSpeed.js

```diff
@@ -229,28 +229,30 @@ export default function DashboardPageSpeed() {
 			const allAudits = select(
 				MODULES_PAGESPEED_INSIGHTS
 			).getAuditsWithStackPack( referenceURL, strategy, 'wordpress' );
+
 			if ( ! allAudits || ! Object.keys( allAudits ).length ) {
 				return [];
 			}
 
-			const audits = [];
-			Object.keys( allAudits ).forEach( ( auditSlug ) => {
-				const audit = allAudits[ auditSlug ];
-				if (
-					( audit.scoreDisplayMode !== 'numeric' &&
-						audit.scoreDisplayMode !== 'binary' ) ||
-					audit.score >= 0.9
-				) {
-					return;
-				}
-
-				audits.push( {
-					id: audit.id,
-					title: audit.title,
-				} );
-			} );
-
-			return audits;
+			return Object.keys( allAudits )
```

Comment:
On a predominantly stylistic note, I'd go with `Object.entries()` here over `Object.keys()`, as it will cut down the line noise of having to dereference `allAudits` in the function bodies, and you can destructure `id` and `title` in the `map()` handler for additional brevity.

You may have to disable the `no-unused-vars` ESLint rule in a couple of spots, but I think it's a worthwhile trade. I'll leave it with you though, if you prefer to keep as-is it's also fine.

Reply:
Good thought, thank you for sharing, @techanvil.

What do you think about using `Object.values()` instead? Example:

```js
Object.values( allAudits )
	.filter(
		( { scoreDisplayMode, score } ) =>
			scoreDisplayMode === 'metricSavings' && score < 1
	)
	.sort( ( a, b ) => {
		// If the scores are the same, sort alphabetically by
		// audit slug. This is how the API returns audits.
		if ( a.score === b.score ) {
			return a.id < b.id ? -1 : 1;
		}

		return a.score - b.score;
	} )
	.map( ( { id, title } ) => ( {
		id,
		title,
	} ) );
```

Reply:
@nfmohit that sounds good, I thought we might still need the keys as audit slugs, but if they're available as the `id` property then `values()` is the way to go!

Reply:
Applied, thank you!